pod-ext/js/constants.js

```diff
@@ -0,0 +1,7 @@
+const constants = Object.freeze({
+	ORIGIN: "http://localhost:3001",
+	REMOTE: "POD_BROWSER",
+	HOST: "POD_EXT",
+	REGISTER: "REGISTER",
+	CHALLENGE: "ASK_CHALLENGE",
```

Comment:
Similarly, this one:

`{ "msg" : "sign_challenge", "challenge" : "AAADEADBEEF" }`